app/views/carts/_cart.html.erb

```diff
@@ -0,0 +1,9 @@
+<div id="<%= dom_id cart %>">
+  <h2 class="font-bold text-lg mb-3">Your Pragmatic Cart</h2>
+
+  <ul class="list-disc list-inside">    
```

Comment:
Let's change this from a `<ul>` to a table so that we can neatly format a total for the order at the bottom of the line items.

Reply:
✅